### ISSUE TITLE
Rename theme options to options

### ISSUE THREAD

**Issue by Xilonz** [htmlburger/carbon-fields#462]
I found out that the theme options can also be used in plugins, so it would make sense to rename it to just 'options' or 'options page'.

**Comment by dmhendricks** [comment]
I believe that it is a throwback that is named as such for compatibility. They would likely alias it rather than rename it, however, it is not something that is visible to end users. I do agree that it is a misnomer.

**Comment by Xilonz** [comment]
Yes, they could also rename the theme options to options and extend the Theme_Options_Container and Network_Options_Container on the Options_Container class, maybe even create an Plugin_Options_Container.

**Comment by dmhendricks** [comment]
Along the lines of what @Xilonz said, it would extremely useful to be able to create `network_options` containers. I'm not sure if that is better served with a separate type (purely as example, `network_options` vs `theme_options`/`site_options` - the particular naming isn't important), or to just have a generic `theme_options`/`options` and modify scope with a method such as `->set_scope( 'network' )` or `->set_scope( 'site' )`. (again, just examples)

Obviously, this addition would have an impact on where the option key/value pairs are stored in the database.

**Comment by Xilonz** [comment]
There already is a [network container](https://github.com/htmlburger/carbon-fields/blob/master/core/Container/Network_Container.php), if you were looking for that.

**Comment by atanas-angelov-dev** [comment]
Just as an FYI, as @Xilonz mentioned, there is a network container but it's available in the latest beta only.

**Comment by dmhendricks** [comment]
Excellent! Personal request: it would be nice if the release would include the `assets/dist` folder. Currently, I have to lock to the specific revision in Composer (else have my changes blown away with the next `composer update`) and copy the `dist` folder manually, else CF doesn't work. (perhaps I'm requiring it incorrectly?? I am not a Composer expert.)

I wish that I would have asked a couple of days earlier! I spent those days hacking a different framework to function in Network admin (totally embarrassing hacks, but it did what I needed it to do, and everything is always due yesterday, of course. I have now converted that code to use CF containers, and it is working well!)

Thank you,
Daniel

**Comment by Xilonz** [comment]
@dmhendricks You dont need to include the asset/dist folder? Just use this [quickstart](https://carbonfields.net/docs/carbon-fields-quickstart/?crb_version=2-1-0). But use ```composer require htmlburger/carbon-fields "2.2.*@dev"``` to get the development version of 2.2, without having to set your composer minimum-stability (which may break other packages)
